client/state/happychat/selectors/test/has-happychat-localized-support.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_CONNECTION_STATUS_UNINITIALIZED,
+	HAPPYCHAT_CONNECTION_STATUS_CONNECTED,
+} from 'state/happychat/constants';
+import hasHappychatLocalizedSupport from 'state/happychat/selectors/has-happychat-localized-support';
+
+describe( '#hasHappychatLocalizedSupport', () => {
+	it( "should be false if there's no active connection", () => {
+		const state = deepFreeze( {
+			happychat: {
+				connection: {
+					status: HAPPYCHAT_CONNECTION_STATUS_UNINITIALIZED,
+					localizedSupport: true,
+				},
+			},
+		} );
+		expect( hasHappychatLocalizedSupport( state ) ).to.be.false;
+	} );
+
+	it( "should be false if Happychat isn't accepting new connections", () => {
```

Comment:
I would also add here that it isn't accepting new localized chat connections or something like that. HC itself could be accepting connections but not for localized chats, or not?

Reply:
That is correct, updated.